templates/withKibana/accountsESDTHistory_policy.go

```diff
@@ -0,0 +1,40 @@
+package withKibana
+
+// AccountsESDTHistoryPolicy will hold the configuration for the accountsesdthistory index policy
+var AccountsESDTHistoryPolicy = Object{
```

Comment:
why are we using snake case in a non-tests file?

Reply:
renamed all the files.

Reply:
renamed all files.